homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -72,6 +72,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# How long to wait for a state change to happen
+STATE_CHANGE_WAIT_TIMEOUT = 2.0
```

Comment:
I'm not sure having separated value makes much difference nor if it's something we want.

One is how long we wait for the state to change (and it may not always change like if we're just changing the contrast). The first one guarantees that the group/light did turn on/off.

The other make sure we trick Alexa long enough to believe we did change things properly as changing attributes like brightness somehow take much longer than changing the state.

The wait for the change is to potentially report an error; but my experience with a real Hue hub is that an error is never reported. Like you're in a room where the power to a light is off: you say Alexa turn the light on and she will say "OK" but nothing happens. Only in the app will you see a banner that the light isn't responding.

As far as Alexa is concerned; only the STATE_CACHED_TIMEOUT value matters, to ensure the GET that follows the PUT gets the same value